app/javascript/mastodon/actions/markers.ts

```diff
@@ -81,7 +84,7 @@ function getLastNotificationId(state: RootState): string | undefined {
     false,
   ) as boolean;
   // eslint-disable-next-line @typescript-eslint/no-unsafe-return
-  return enableBeta
+  return enableBeta || forceGroupedNotifications
```

Comment:
Could we move this to a selector, rather than manually getting the settings everywhere?

Reply:
Done